SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Critical bugs and security issues 💥
+
+If you're here because you're trying to figure out how to notify us of a security issue, go to [Discord](https://discord.gg/VNyAnw7e), and alert:
+
+
+- Ethan Frey `ethanfrey#9693`
+- elsehow `elsehow#3115`
```

Comment:
```suggestion
- Ethan Frey `ethanfrey#9693`
- Simon Warta `Simon | Confio#9061`
```
@webmaster128 

I appriciate you making this PR, but shouldn't recommended people be at least from CosmWasm circle? From issue I assume you're collaborating with @JakeHartnell, I recognize him.
@ethanfrey  what do you think?

Reply:
Yeah, this repo is maintained by Confio, it should be us getting those mails.

Happy if you use this proposal on cw-nfts. And if we spin off maintainership of other repos, happy for them to use other contact methods.

Reply:
I would prefer to not be on the list, all else equal. I added my own name so Ethan wouldn't be the only person on the hook.

If we move forward with this, it would be great to put together a list of folks who should be notified!

Reply:
> this repo is maintained by Confio, it should be us getting those mails

I think this is a crucial point/decision. As long as Confio is the maintainer, we should use security@confio.gmbh as the only security contact as shown in https://github.com/CosmWasm/wasmd/blob/master/SECURITY.md.